ros2topic/ros2topic/api/__init__.py

```diff
@@ -182,20 +186,20 @@ def qos_profile_from_short_keys(
 
 
 def add_qos_arguments_to_argument_parser(
-    parser: argparse.ArgumentParser, is_publisher: bool = True, default_preset: str = 'sensor_data'
+    parser: argparse.ArgumentParser, is_publisher: bool = True,
+    default_preset: str = default_preset_str
```

Comment:
this argument is unused

Reply:
Used [here](https://github.com/gonzodepedro/ros2cli/blob/afb4873b11076b8a1a4f80f01a8c724ae97dc197/ros2topic/ros2topic/api/__init__.py#L199)